src/plugins/template/backend/ops/string_tensor_unpack.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/reference/string_tensor_unpack.hpp"
+
+#include "evaluate_node.hpp"
+#include "string_tensor_unpack_shape_inference.hpp"
+
+template <>
+bool evaluate_node<ov::op::v15::StringTensorUnpack>(std::shared_ptr<ov::Node> node,
+                                                    ov::TensorVector& outputs,
+                                                    const ov::TensorVector& inputs) {
+    if (node->get_input_element_type(0) == ov::element::string) {
+        auto string_tensor_unpack = std::dynamic_pointer_cast<ov::op::v15::StringTensorUnpack>(node);
+        std::vector<ov::PartialShape> output_shapes;
+        if (string_tensor_unpack) {
```

Comment:
Use OPENVINO_ASSERT instead this `if` block?

Reply:
Done.